src/redux/selectors/data/categoriesSelectors.js

```diff
@@ -25,3 +25,14 @@ export const selectSearchGroup = createSelector(
     return searchGroup
   }
 )
+
+export const selectSearchGroupBySearchResult = createSelector(
+  selectSubcategoriesAndSearchGroups,
+  (state, offerFromResult) => offerFromResult.searchGroupName,
+  (categories, searchGroupName) => {
+    if (!searchGroupName) return {}
+    const { searchGroups } = categories[0]
+    const searchGroup = searchGroups.find(searchGroup => searchGroup.name === searchGroupName)
+    return searchGroup
```

Comment:
Question de préférence mais j'aurais inliner la variable de retour, puisque la fonction est bien nommée, on veut un searchGroup et on cherche dans les searchGroups ça me parait assez explicite
```suggestion
    return searchGroups.find(searchGroup => searchGroup.name === searchGroupName)
```

Reply:
:+1: